spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -239,6 +239,24 @@
 				<artifactId>spring-cloud-spanner-spring-data-r2dbc</artifactId>
 				<version>${project.version}</version>
 			</dependency>
+			<dependency>
+				<groupId>com.google.cloud</groupId>
+				<artifactId>spring-cloud-gcp-starter-alloydb</artifactId>
+				<version>${project.version}</version>
+			</dependency>
+
+			<!-- spring-cloud-gcp-starter-alloydb -->
+			<dependency>
+			    <groupId>com.google.cloud</groupId>
+			    <artifactId>alloydb-jdbc-connector</artifactId>
+			    <version>1.1.1</version>
```

Comment:
Can you extract this into a property at the top like the other dependencies?

Reply:
updated.